### PR TITLE
Fix presence test

### PR DESCRIPTION
**Pull request by ruudniew**
Sometimes a notification is delivered, which fails the test. This should fix it.

### CODE REVIEW

File: test/integration/presence.js [hunk 1]
```diff
@@ -135,17 +135,19 @@ describe('Presence', function () {
...
-					// expect to see thierry join, james join and james leave
-					if (results.length === 3) {
-						const expected = [
-							[1, 'user1'],
-							[2, 'james'],
-							[1, 'james'],
-						];
-						expect(results).to.deep.equal(expected);
-						resolve();
+					if (e.type.split('.')[0] !== 'notification') {
```
Comment (gumuz) [inline test/integration/presence.js:138]: `!(e.type.startsWith('notification.'))` might be a bit more readable

Comment (gumuz) [reply test/integration/presence.js:138]: but wouldn't it be nicer to instead subscribe or check for the specific event we're waiting for?

Comment (AnatolyRugalev) [reply test/integration/presence.js:138]: I agree. Seems strange to filter out events this way. At least we could reverse the condition and make it return early to make it simpler to read and understand

**Review by gumuz — APPROVED** [review]
LGTM!

**Pull request merged**